config.yaml

```diff
@@ -178,6 +178,15 @@ commands:
 
   N98\Magento\Command\Installer\InstallCommand:
     magento-packages:
+      - name: magento-ce-1.9.3.4
```

Comment:
please remove this section, the namespace is for original packages that get mirrored, not the open mage magento mirror.